src/planner/match/MatchSolver.h

```diff
@@ -30,7 +30,7 @@ class MatchSolver final {
 
     static Expression* rewrite(const LabelAttributeExpression* la);
 
-    static Expression* doRewrite(const std::unordered_map<std::string, AliasType>& aliases,
+    static Expression* doRewrite(const std::unordered_map<std::string, AliasType>* aliases,
```

Comment:
Why replace it with ptr? i don't see any improvement.

Reply:
eg. wctx->aliases might be a nullptr, so I need add DCHECK(wctx->aliases) before call doRewrite, and  everytime I call doRewrite I have to add this DCHECK. 
So I replace this argument with ptr, and so I can add DCHECK in the function body of doRewrite
```
    auto rewriter = [wctx](const Expression* expr) {
        return MatchSolver::doRewrite(*wctx->aliases, expr);
    };
```